app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTaskView.kt

```diff
@@ -4,23 +4,29 @@ import android.content.Context
 import android.content.res.ColorStateList
 import android.util.AttributeSet
 import android.view.View
-import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.cardview.widget.CardView
 import androidx.core.widget.ImageViewCompat
 import kotlinx.android.synthetic.main.view_suggested_edits_task_item.view.*
 import org.wikipedia.Constants.MIN_LANGUAGES_TO_UNLOCK_TRANSLATION
 import org.wikipedia.R
 import org.wikipedia.WikipediaApp
+import org.wikipedia.util.DimenUtil
 import org.wikipedia.util.ResourceUtil
 
-internal class SuggestedEditsTaskView constructor(context: Context, attrs: AttributeSet? = null) : ConstraintLayout(context, attrs) {
+internal class SuggestedEditsTaskView constructor(context: Context, attrs: AttributeSet? = null) : CardView(context, attrs) {
 
     init {
+        val params = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+        params.topMargin = DimenUtil.roundedDpToPx(8f)
+        params.bottomMargin = params.topMargin
+        params.leftMargin = DimenUtil.roundedDpToPx(16f)
+        params.rightMargin = params.leftMargin
+        layoutParams = params
+        setCardBackgroundColor(ResourceUtil.getThemedColor(getContext(), R.attr.paper_color))
```

Comment:
Not sure if it is the expected design, but the cards can be barely seen in the dark/black mode.

Reply:
Let's let Robin comment on it during his review.